src/app.js

```diff
@@ -225,7 +225,8 @@ const CreateKeyboard = (lang, shift) => {
   localStorage.setItem('KeysLang', keysLang);
   document.querySelector('.keyboard').innerHTML = '';
   let key;
-  for (let i = 0; i < lang.length; i += 1) {
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < lang.length; i++) {
```

Comment:
You can fix this rule in elint config
Or better to use array functions like forEach.